src/Compilers/CSharp/Portable/Binder/Binder_Expressions.cs

```diff
@@ -2997,6 +3012,8 @@ internal GlobalExpressionVariable LookupDeclaredField(SyntaxNode node, string id
             bool hasNames = result.Names.Any();
             if (nameColonSyntax != null)
             {
+
```

Comment:
These two added empty lines (line 3015) are unnecessary

Reply:
Done.  NOte: we have an analzyer for this :)  It would be great to turn it on in the compiler layer. :)